Hunting Queries/MultipleDataSources/AzureResourceCreationWithNetworkActivity.yaml

```diff
@@ -0,0 +1,99 @@
+id: ac25d05d-362d-4a8d-b4e7-58c0edd2379c
+name: Anomalous Resource Creation and related Network Activity
+description: |
+  'Indicates when an anomalous number of resources are created successfully in Azure via the AzureActivity log.
```

Comment:
Maybe add a why to the description as well as the what. If this fires an alert it might be useful for the analyst to understand why this is bad.

Reply:
It is hunting, but will think on a bit more details in the description